### DISCUSSION TITLE
can it get me banned

### DISCUSSION THREAD

**Discussion by Ayush-Mann-0** [cafali/SnapKey#25] [Q&A]
im sure it wouldnt get me banned on CS but im not too sure about valorant as it can be considered as cheating, did you try it on something other than CS

**Comment by Yaw-Dev** [comment]
I tried it on Valorant for a while (2 competitive matches) and nothing happened so I guess it's fine.

**Comment by Ayush-Mann-0** [comment]
alright thanks

**Reply by kartoofy** [reply]
Have you been using it?

**Reply by Ayush-Mann-0** [reply]
> Have you been using it?

i did use it , but in valorant its barely useful , so yeah

**Comment by toinouH** [comment]
https://github.com/cafali/SnapKey/discussions/112 Someone claims to be banned using this. I don't know if Riot dislike it or not.

**Comment by retr0dedsec03** [comment]
I got banned from Valorant using this… that too permanent ban 🫠
